repos/fdbt-site/src/pages/salesConfirmation.tsx

```diff
@@ -113,21 +156,25 @@ const SalesConfirmation = ({
     </TwoThirdsLayout>
 );
 
-export const getServerSideProps = (
+export const getServerSideProps = async (
     ctx: NextPageContextWithSession,
-): GetServerSidePropsResult<SalesConfirmationProps> => {
+): Promise<GetServerSidePropsResult<SalesConfirmationProps>> => {
     const fareTypeAttribute = getSessionAttribute(ctx.req, FARE_TYPE_ATTRIBUTE);
     if (!fareTypeAttribute) {
         return { redirect: { destination: '/home', permanent: false } };
     }
     const csrfToken = getCsrfToken(ctx);
     const salesOfferPackageInfo = getSessionAttribute(ctx.req, SALES_OFFER_PACKAGES_ATTRIBUTE);
     const ticketDatingInfo = getRequiredSessionAttribute(ctx.req, PRODUCT_DATE_ATTRIBUTE);
+    const fullCaps = (getSessionAttribute(ctx.req, CAPS_DEFINITION_ATTRIBUTE) as CapsDefinition) || null;
+    const nocCode = getAndValidateNoc(ctx);
+    const caps = await getCaps(nocCode);
```

Comment:
This always returns an array, even if its just an empty array, which means we dont need it to be optional in the props.